core/src/main/java/hudson/model/Queue.java

```diff
@@ -1509,8 +1533,20 @@ public Api getApi() {
          * VM-wide unique ID that tracks the {@link Task} as it moves through different stages
```

Comment:
That was from the original author. I can update it if you think that's clearer (and it seems like it is).